src/tools/fs_utils.rs

```diff
@@ -158,10 +158,14 @@ fn main() {
 
         "format" => {
             lind_deltree(&utilcage, "/"); //This doesn't actually fully remove all of the linddata files... TODO: debug
-            let mut metadata = FS_METADATA.write().unwrap();
-            *metadata = FilesystemMetadata::blank_fs_init();
-            drop(metadata);
-            load_fs_special_files(&utilcage);
+
+            let mut logobj = LOGMAP.write().unwrap();
+            let log = logobj.take().unwrap();
+            let _close = log.close().unwrap();
+            drop(logobj);
+
+            format_fs();
```

Comment:
I meant here, we dont explicilty delete the log? This was probably my error initially...

Reply:
I don't entirely understand what you are saying. I believe you are saying we don't remove the log here. Yes that is true, but because we continue on we overwrite the log file in create_log and it ultimately gets removed at the end if it exits correctly. Because you want me to exit rather than carry on then yes I will have to remove it.

Reply:
Oh ok now I see what you're saying

Reply:
Also wait I'm just confused we already do remove the log in format_fs